README.md

```diff
@@ -1,9 +1,15 @@
 # `Discovery`
 
-Discover the world of microcontrollers through
-[Rust](https://www.rust-lang.org/):
+Discover the world of microcontrollers through [Rust](https://www.rust-lang.org/)!
 
-- Read this book on-line at https://docs.rust-embedded.org/discovery/
+There are currently two versions of this book. The first is older and uses an
+F3 Discovery circuit board to introduce you to microcontrollers and Rust, while
+the second is newer and uses a micro:bit circuit board instead.
```

Comment:
this new book does teach a bit about microcontrollers, so maybe that should be mentioned.